webapp/channels/src/components/interactive_dialog/interactive_dialog.test.tsx

```diff
@@ -41,7 +48,7 @@ describe('components/interactive_dialog/InteractiveDialog', () => {
             };
             const wrapper = shallow(<InteractiveDialog {...props}/>);
 
-            await wrapper.instance().handleSubmit({preventDefault: jest.fn()});
+            await (wrapper.instance() as InteractiveDialog).handleSubmit(submitEvent);
```

Comment:
I prefer to use `shallow<InteractiveDialog>(`, so that way you can remove the `as InteractiveDialog` here. And of course, the same for all the other cases in this file.